comments/templates/comments/comments.html

```diff
@@ -27,7 +27,7 @@
                                 <input type="hidden" name="message">
                             </div>
                             <div class="comment-visible-fields">
-                                <textarea class="{% block post_textarea_class %}{% endblock post_textarea_class %}" name="message_holder"></textarea>
+                                <textarea id="{% block textarea_id %}comment-{{ node.id }}{% endblock textarea_id %}" class="{% block post_textarea_class %}{% endblock post_textarea_class %}" name="message_holder"></textarea>
```

Comment:
I don't think `comment-` is good here since it will always have a handing hyphen.  How about id='new-comment' or something like that